lib/ui/home.dart

```diff
@@ -49,10 +55,15 @@ class _HomeState extends State<Home> with WidgetsBindingObserver {
 
   final List<Widget> _children = [
     PlotList(),
-    HomeProfitLossChild(),
+    ProfitLossPage(),
     ArticleList(),
     HomeCommunityChild(),
-    PlaygroundView(widgetList: [], appBarColor: Color(0xFF9CBD3A),),
+    PlaygroundView(widgetList: [
```

Comment:
@jessicahernandezhv moved it to a separated file. Do the same with the same file name (you will have conflicts)